pyupgrade.py

```diff
@@ -1101,7 +1102,17 @@ def _visit_func(self, node):
         else:
             self.generic_visit(node)
 
-    visit_FunctionDef = visit_Lambda = _visit_func
+    def _visit_sync_func(self, node):
+        self._in_async_def, orig = False, self._in_async_def
+        self._visit_func(node)
+        self._in_async_def = orig
+
+    visit_FunctionDef = visit_Lambda = _visit_sync_func
+
+    def visit_AsyncFunctionDef(self, node):
```

Comment:
you'll need to `# pragma: no cover PY35+` this function since it's version dependent